src/shared/extension-release.c

```diff
@@ -0,0 +1,79 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
```

Comment:
what's the reason for moving this? it checks compat between os-release and extension-release both of which are implemented by os-release.[ch], sounds close enough to me

Reply:
It's moved out of os-util so that it stays in shared/. This has a lot of messages and such and I think it fits better in shared/.